PhpAmqpLib/Wire/IO/StreamIO.php

```diff
@@ -276,7 +276,18 @@ public function select($sec, $usec)
         $write = null;
         $except = null;
 
-        return stream_select($read, $write, $except, $sec, $usec);
+        $result = @stream_select($read, $write, $except, $sec, $usec);
```

Comment:
If you look at the error handler it ignores the case you are talking about. The pull request is for fixing signals and the first few changes that I did using @ exactly like you have were not used.